ckan/tests/controllers/test_user.py

```diff
@@ -769,22 +769,6 @@ def test_request_reset_by_name(self, send_reset_link, app):
         assert "A reset link has been emailed to you" in response
         assert send_reset_link.call_args[0][0].id == user["id"]
 
-    @mock.patch("ckan.lib.mailer.send_reset_link")
-    def test_request_reset_when_duplicate_emails(self, send_reset_link, app):
-        user_a = factories.User(email="me@example.com")
-        user_b = factories.User(email="me@example.com")
-
-        offset = url_for("user.request_reset")
-        response = app.post(
-            offset, data=dict(user="me@example.com")
-        )
-
-        assert "A reset link has been emailed to you" in response
-        emailed_users = [
-            call[0][0].name for call in send_reset_link.call_args_list
-        ]
-        assert sorted(emailed_users) == sorted([user_a["name"], user_b["name"]])
-
```

Comment:
What's wrong with this test?

Reply:
This test is currently deleted from core, since we implemented validator for email uniqueness